commerce-frontend-taglib/src/main/java/com/liferay/commerce/frontend/taglib/servlet/taglib/AddToCartTag.java

```diff
@@ -69,12 +71,20 @@ public int doStartTag() {
 
 			CommerceOrder commerceOrder = commerceContext.getCommerceOrder();
 
+			int productOrderQuantity = 0;
+
 			if (commerceOrder != null) {
 				putValue("orderId", commerceOrder.getCommerceOrderId());
+
+				List<CommerceOrderItem> commerceOrderItems =
```

Comment:
@alecsloan  I am mergin this, but put in the backlog a change to use count from service

Reply:
@marco-leo @gianmarcobrunialti See my diff on pull #1348 for implementing new methods to get OrderItems by CPInstanceId